deployment-configuration.md

```diff
@@ -1,25 +1,27 @@
 Deployment Configuration
 ========================
 
-This document decribes how to install the software on various platforms.
+This document describes how to install the software on various platforms.
 
 Run on Amazon Web Services EC2 using an Amazon Machine Image (AMI)
 ==================================================================
```

Comment:
Having a bunch of H1's make this a little hard to read.  You should use H2's instead (`------------------------`)

Reply:
(and same for all sections below)